src/platforms/flutter/usage/advanced-usage.mdx

```diff
@@ -4,6 +4,9 @@ sidebar_order: 2
 description: "Learn about advanced uses of the Flutter SDK, including uploading debug symbols, known limitations, and bundling."
 ---
 
+## Requirements
+
+- For the usage of the Flutter SDK, the minimal required Dart SDK version is `2.12.0` and Flutter SDK version is `1.17.0`
```

Comment:
you probably need to pull the master into this branch, this advanded-page does not exist anymore for the Flutter docs, we can place it somewhere else

Reply:
see https://docs.sentry.io/platforms/flutter/ here the version requirement now is under the feature list.
so you have to edit this list with the correct version https://github.com/getsentry/sentry-docs/blob/master/src/includes/getting-started-primer/flutter.mdx